staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -250,6 +250,15 @@ type ObjectMeta struct {
 	// is an identifier for the responsible component that will remove the entry
 	// from the list. If the deletionTimestamp of the object is non-nil, entries
 	// in this list can only be removed.
+	// Finalizers may be processed and removed in any order.  Order is NOT enforced
+	// because it introduces significant risk of stuck finalizers.
+	// finalizers is a shared field, any actor with permission can reorder it.
```

Comment:
```suggestion
	// finalizers is a shared field; any actor with permission can reorder it.
```
Should the leading 'f' be capitalized? Is this documenting the Go field name, or the JSON field name?

Reply:
documentation should reference the API field, so lowercase `f`